source/docs/helpers.md

```diff
@@ -139,11 +139,18 @@ Loads CSS files. `path` can be an array or a string. If `path` isn't prefixed wi
 <%- css(['style.css', 'screen.css']) %>
 // <link rel="stylesheet" href="/style.css">
 // <link rel="stylesheet" href="/screen.css">
+
+<%- css({ href: 'style.css', integrity: 'foo' }) %>
+// <link rel="stylesheet" href="/style.css" integrity="foo">
+
+<%- css([{ href: 'style.css', integrity: 'foo' }, { href: 'screen.css', integrity: 'bar' }]) %>
```

Comment:
Should we add more example about attributes? For example, `onload` or `media`.

Reply:
the existing example is sufficient, there's no indication that there is a limit (there's no limit on number of attributes, anyway).